### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Agrover112**
Critical Spelling errors in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,11 +30,12 @@ Each resource is formated as a tuple of **(description, link, type, contributor)
  - People/Labs/Organisations
  - Opportunities/Jobs
  - Conferences/Workshops/Talks
-- **contributor** is the name of the person who contributed the resource.
+- **contributor** is the name of the person who has contributed to the resource.
```
Comment (rishistyping) [inline README.md]: Revert to original language.

**Review by rishistyping — COMMENTED** [review]
Change

File: README.md (hunk 1 above)
Comment (Agrover112) [reply README.md]: Proper grammar is necessary

Comment (rishistyping) [reply README.md]: The original is the proper lang.

**Comment by Agrover112** [comment]
Should be updated now

**Pull request merged**